tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -130,6 +130,10 @@ TILEDB_EXPORT int32_t tiledb_array_schema_evolution_drop_attribute(
 
 /**
  * Sets timestamp range in an array schema evolution
+ * The evolved schema could have the same timestamp as previous schema.
```

Comment:
I think we need to edit this a little bit because I still find this description confusing. Suggest the following:
- remove the sentence `the evolved schema could have ...`
- add:
```
This function sets the output timestamp of the committed array schema after evolution.
The lo and hi values are currently required to be the same or else an error is thrown.
```